### PR TITLE
Squash the modal when running behat test

### PR DESCRIPTION
**Pull request by maxime-rainville**
This PR deactivate the modal when running behat test

# Parent issue
* https://github.com/silverstripeltd/open-sourcerers/issues/68

### CODE REVIEW

**Review by dnsl48 — DISMISSED** [review]
LGTM

File: .travis.yml [hunk 1]
```diff
@@ -25,6 +25,7 @@ env:
     - XVFBARGS=":99 -ac -screen 0 1024x768x16"
     - SS_BASE_URL="http://localhost:8080/"
     - SS_ENVIRONMENT_TYPE="dev"
+    - SUPPRESS_MODAL=1
```
Comment (dnsl48) [inline .travis.yml]: Oh, actually, maybe let's rename it to `SS_BAMBUSA_SUPPRESS_MODAL`

**Review by dnsl48 — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 1 above)
Comment (maxime-rainville) [reply .travis.yml]: Done.

**Review by dnsl48 — APPROVED** [review]
LGTM, merge on green :)

**Pull request merged**